dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java

```diff
@@ -80,6 +111,52 @@ protected boolean isGithubAppEnabledOnRepository(String fullRepoName) {
         }
     }
 
+    /**
+     * Method to verify whether the github app is installed on a repository or not, using Renovate API
+     * @param fullRepoName = The repository full name, i.e, of the format "owner/repoName". Eg: "Salesforce/dockerfile-image-update"
+     * @return True if github app is installed, false otherwise. 
+     * Reference: https://github.com/mend/renovate-ce-ee/blob/main/docs/reporting-apis.md#repo-info
+     */
+    protected boolean isGithubAppEnabledOnRepositoryWithRenovateApi(String fullRepoName) {
+        try {
+            String apiEndpoint = "https://scot-mend-renovate072.sfproxy.buildndeliver.aws-esvc1-useast2.aws.sfdc.cl/api/job/repos/%s";
```

Comment:
Let us not expose this URL in the internet. Read this from properties or the CLI.

Reply:
> QQ: Did we try with JUST retries around the GithubAppEnabledOnRepository code? I have a gut feeling just the retries will eliminate the errors to a large large extent.
According to [the WI](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000020EavoYAC/view), part of the retry mechanisms is if after few retries it doesn't work, it would utilize the Renovate API

Reply:
> Let us not expose this URL in the internet. Read this from properties or the CLI.
I will surely mask that with SFCI credentials